tool/README.md

```diff
@@ -202,19 +184,12 @@ From the git GUI tool or from github.com directly:
    flutter pub publish
    ```
 
-### Write release notes for the release
-Release notes should contain details about the user-facing changes included in the release.
-
-These notes are shown directly in DevTools when a user opens a new version of DevTools.
-
-1. Request the team to verify their important user facing changes are documented in
-   [release-notes-next.md][1].
-
-2. See the release notes
-[README.md](https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/framework/release_notes/README.md)
-for details on where to add DevTools release notes to Flutter website and how to test them.
-
-3. Copy the content of [release-notes-template.md](../packages/devtools_app/lib/src/framework/release_notes/release-notes-template.md) to [release-notes-next.md][1], to contain
-   draft for the next release.
+### Push the DEVTOOLS_NEXT_BRANCH
+```shell
+git checkout $DEVTOOLS_NEXT_BRANCH
+git push -u origin $DEVTOOLS_NEXT_BRANCH
```

Comment:
will we need to merge master here?

Reply:
it is already done as part of the script. the branch should be fresh as a daisy :)

Reply:
but after we merge the clean branch to master, only then will we be creating a PR for this next branch and merging, so at that point, it would be one commit behind

Reply:
the 2nd branch is built off of the first, so I don't think we will have an issue with this.
If it does end up being an issue then I will come back and modify this process or the instructions :)